cumulusci/cumulusci.yml

```diff
@@ -269,6 +269,12 @@ tasks:
         options:
             seconds: 5
 
+    info:
```

Comment:
Can we change this task name to log instead?  That way you could call log and override the level via options